### DISCUSSION TITLE
Fermata Auto Icon

### DISCUSSION THREAD

**Discussion by ahhl** [AndreyPavlenko/Fermata#427] [Ideas]
just wondering whether fermata icon can indicate whether its fermata auto or fermata control or fermata mirror of FS mirror as in my car the the icon word are being truncated. all can see is fermata with similar icon. perhaps can create small name on top of fermata auto of what it is. ya i am aware the mirror version are of reddish color, but still cannot differentiate between fs mirror or mirror

**Comment by AndreyPavlenko** [comment]
I can rename Fermata FS Mirror to FS Fermata Mirror

**Comment by cookiehome** [comment]
![IMG_20240315_070614](https://github.com/AndreyPavlenko/Fermata/assets/155454118/b32b9aef-5377-459e-a0f5-781e2850aac8)
cannt read any wording. only icon. wird truncated
